docs/application-developers/backuppolicy.md

```diff
@@ -0,0 +1,66 @@
+You can retain backups based on Count which will delete all the backups exciding the number specified, or define Retention Period specifies the maximum age of backups to retain e.g. 7 (days). Backups older than this period will be deleted. This period is calculated based on the creation timestamp of the backup. It supports multi duration types: `hours`, `days`, `months` and `years`.
+
+> More features will be added to the policy in future such as `Compression`, `Notifications`, `Encryption`
+
+## Configuration
+`BackupPolicy` consists of different configuration that manages the backup scheduling and other features.
+
+### ScheduleConfiguration
+The ScheduleConfiguration defines the schedule type to use, we support for now Cron configuration which is the most common used method for creating schedules. More types could be supported in future based on the user usecases.
+The config gives the ability to specify timezone to calculate the schedule based upon, the default is `UTC`.
+*Sample*
+
+```yaml
+scheduleConfig:
+    scheduleType: Cron
+    timezone: "CET"
+    Cron:
+        expression: "0 0 * * *"
+```
+
+### RetentionConfiguration
+The `RetentionConfiguration` defiones the policy for retaining and deleting old backups. It ensure that the backups are kept for a specified duration or up to a specified number of backups.
+
+#### Retention by Count
+When using retention by count, you specify the maximum number of backups to keep. Once this limit is exceeded, the oldest backups are deleted based on `creationTimestamp`
+
+*Sample*
+```yaml
+spec:
+    retentionConfig:
+        count: 5
+```
+
+#### Retention by Period
+When using Period for retention, you specify count and type. Type is the duration type a backup exceeding the count specified will be deleted starting from the `creationTimestamp` i.e. 5 `days`, 2 `month` or even 3 `years`
+
+*Sample*
+```yaml
+spec:
+    retentionConfig:
+        retentionPeriod:
+            type: "days"
+            count: 15
+```
+
+In the above sample, the controller will fetch the backups created by the policy, check the createdTimestamp with 15 days, if it exceeds current date, backups will be marked for deletion.
+
+## BackupPolicy Controller 
+The newly created policy, will create a backup immediately. A backup is created if there is no backups or if the scheduling time elapsed.
+
+After creating a policy, you can disable it, which allow you to keep it instead of just deleting the resource, however, updating the status and reconcile will not be possible until enabling it again. All policies created are enabled by default.
+
+The backup created will have a generated name with the convention `backup-`, where each backup created will be labeled by key `a8s.a9s/policy`  and value as the policy name.
+
+Backup will be created without checking if there is a running backup. We need to validate the case of long running backups. 
+> Follow up ticket https://anynines.atlassian.net/browse/A8S-2451
```

Comment:
I don't think it makes sense to include this ticket in the user facing documentation. This could be included in the component repository for developers.